plugins/modules/database/mysql/mysql_replication.py

```diff
@@ -28,7 +28,7 @@
       C(getslave) (SHOW SLAVE STATUS),
       C(startslave) (START SLAVE),
       C(stopslave) (STOP SLAVE),
-      C(resetmaster) (RESET MASTER) - supported from Ansible 2.10,
+      C(resetmaster) (RESET MASTER) - supported from community.general 0.2.0 on,
```

Comment:
```suggestion
      C(resetmaster) (RESET MASTER) - supported from community.general 0.2.0,
```

Reply:
Suggestion from @acozine:
```suggestion
      C(resetmaster) (RESET MASTER) - supported since community.general 0.2.0,
```